core/src/main/java/org/testcontainers/images/builder/ImageFromDockerfile.java

```diff
@@ -128,9 +132,15 @@ public void onNext(BuildResponseItem item) {
                     Transferable transferable = entry.getValue();
                     final String destination = entry.getKey();
                     transferable.transferTo(tarArchive, destination);
+                    bytesToDockerDaemon += transferable.getSize();
                 }
                 tarArchive.finish();
             }
+            
+            log.info("Transferred {} KB to Docker daemon", bytesToDockerDaemon % 1024);
+            if (bytesToDockerDaemon > 50 * 1024 * 1024) // warn if >50MB sent to docker daemon
+                log.warn("A large amount of data was sent to the Docker daemon ({}). Consider using a .dockerignore file for better performance.", 
+                        bytesToDockerDaemon % (1024 * 1024));
```

Comment:
As above, shouldn't it be?
```suggestion
                        bytesToDockerDaemon / (1024 * 1024));
```
and add `MB` to the log message so that it's clear.

Reply:
yes, good catch!